dali/test/python/operator_1/test_crop.py

```diff
@@ -245,7 +245,7 @@ def test_crop_no_cast_vs_cast_to_float_and_back():
 
 class Crop3dPipeline(Pipeline):
     def __init__(self, device, batch_size, iterator, data_shape, data_layout, num_threads=1,
-                 device_id=0, crop_seq_as_depth=False):
+                 device_id=0, crop_seq_as_depth=False, fixed_window=False):
```

Comment:
Is it really "fixed window" or rather "separate crop dimensions"?

Reply:
Done